tools/memap.py

```diff
@@ -9,6 +9,7 @@
 import json
 import argparse
 from prettytable import PrettyTable
+from StringIO import StringIO
```

Comment:
This was used to allow the csv exporter to write to a string (so it can be returned). The way it is currently written there is no way to get the output as a string before writing it to a file. This may now actually be a big deal though, so I will revisit this.